tests/test_arrange.py

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
I believe only the zip archive should be committed not the unzip folders.

Reply:
You have committed the unzip archive as well AFAICT. The following is part of the repo.
tests/assets/valid_dataset/dicom_nested_one_level/subdir_0/image_1.dcm 
I think that is not requried and only the zip version needed to be committed? Maybe I am mistaking.

Reply:
Non sense from me. All good. Resolving